openvdb/openvdb/tools/VolumeToMesh.h

```diff
@@ -1614,10 +1614,11 @@ ComputePoints<InputTreeType>::operator()(const tbb::blocked_range<size_t>& range
             if (inclusiveCell) getCellVertexValues(*inputNode, offset, values);
             else               getCellVertexValues(inputAcc, ijk, values);
 
-            size_t count;
+            size_t count, weightcount;
 
             if (refSigns == 0) {
                 count = computeCellPoints(points, values, signs, iso);
+                weightcount = 0;
```

Comment:
Minor: maybe instead just do `weightedPointMask.fill(false)` ? as the `weightcount = count` in the second branch is confusing (essentially all we want to know is if `weightedPointMask` is initialized).

Reply:
My logic was to mimic the original code - ie, we have the idea that weightcount is actually a vector that is being filled in. 

I would claim count itself is very confusing in this code - I much prefered the old code with the count tied to the vectors....  Internally we have UT_SmallArray that allows stack-based vectors with a size....

Reply:
It made more sense when it was a vector yeah, a smallarray impl would be good but probably overkill even here when the max size is alwasy 4. Anyway this was a minor thing